src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java

```diff
@@ -27,6 +27,11 @@ public LeaseClientProvider getLeaseClientProvider() {
         return blobClient -> new BlobLeaseClientBuilder().blobClient(blobClient).buildClient();
     }
 
+    @Bean("storage-client")
+    public BlobServiceClient getStorageClient(StorageSharedKeyCredential credentials) {
```

Comment:
there are 3 storage clients now ?

Reply:
yes. `storage-client` which is service's own one. and 2 prefixed ones noting what are they